snscrape/modules/vkontakte.py

```diff
@@ -347,9 +347,11 @@ def _get_entity(self):
 			if websites:
 				kwargs['websites'] = websites
 
-		def parse_num(s):
+		def parse_num(s: str) -> int:
```

Comment:
The return value is a tuple of two ints, so this should be `typing.Tuple[int, int]` (`tuple[int, int]` is only supported in Python 3.9+).

Reply:
Yeah, I'll make the changes. Admittedly the precision stuff went a bit over my head; we haven't been using it.